modules/virt-configuring-secondary-dns-server.adoc

```diff
@@ -55,7 +55,7 @@ metadata:
   name: kubevirt-hyperconverged
   namespace: {CNVNamespace}
 spec:
-    featureGates:
+  featureGates:
       deployKubeSecondaryDNS: true
     kubeSecondaryDNSNameServerIP: "10.46.41.94" <1>
```

Comment:
same with these two lines - 2 extra space should be removed:

```
spec:
  featureGates:
    deployKubeSecondaryDNS: true
  kubeSecondaryDNSNameServerIP: "10.46.41.94" <1>
```

Reply:
```
@dshchedr Should 
deployKubeSecondaryDNS: true
  kubeSecondaryDNSNameServerIP:
```
both be in line with` featureGates`?

Reply:
no, deployKubeSecondaryDNS - this one inside featureGates (4 spaces)
kubeSecondaryDNSNameServerIP - in line with featureGates (2 spaces)

```
spec:          <-- 0                                                                                       
  featureGates:       <-- 2
    deployKubeSecondaryDNS: true       <-- 4
  kubeSecondaryDNSNameServerIP: "10.46.41.94"       <-- 2
```

Reply:
Thank you Denys for the comment. @sabrinajess You need to update as Denys pointed out in the comment above ^